frappe/model/delete_doc.py

```diff
@@ -198,7 +198,10 @@ def check_if_doc_is_linked(doc, method="Delete"):
 			for item in frappe.db.get_values(link_dt, {link_field:doc.name},
 				["name", "parent", "parenttype", "docstatus"], as_dict=True):
 				linked_doctype = item.parenttype if item.parent else link_dt
-				if linked_doctype in doctypes_to_skip:
+
+				ignore_linked_doctypes = doc.get('ignore_linked_doctypes') or []
```

Comment:
There is already a flag `ignore_links` used in document.py for the same purpose, you should use the same.
https://github.com/frappe/frappe/blob/develop/frappe/model/document.py#L1009

Reply:
@nabinhait  Initially I used that flag only but ignore_links ignores all the links. I want to ignore only some specific doctypes like GLE and SLE and no other doctypes. Error should be given if linked with any other doctype.